tests/parse/test_parse_case.py

```diff
@@ -54,30 +54,39 @@ def test_parse_case_date(scout_config):
     assert isinstance(case_data["analysis_date"], datetime)
 
 
-def test_parse_case_owner(scout_config):
-    # GIVEN you load sample information from a scout config
-    # WHEN case is parsed
-    case_data = parse_case(scout_config)
-    # THEN the case should have a owner
-    assert case_data["owner"] == scout_config["owner"]
-
-
-def test_parse_case_limsid(scout_config):
-    """Test parsing a case with lims_id"""
-
+@pytest.mark.parametrize(
+    "param_name",
+    [
+        "analysis_date",
+        "delivery_report",
+        "gene_panels",
+        "lims_id",
+        "owner",
+        "peddy_ped",
+        "rank_model_version",
+        "rank_score_threshold",
+        "smn_tsv",
+    ],
+)
+def test_parse_case_parsing(scout_config, param_name):
     # GIVEN you load sample information from a scout config
     # WHEN case is parsed
     case_data = parse_case(scout_config)
-    # THEN the lims ID should be the same as in config
-    assert case_data["lims_id"] == scout_config["lims_id"]
+    # THEN the case should have a the parameter
+    assert case_data[param_name] == scout_config[param_name]
 
 
-def test_parse_case(scout_config):
+@pytest.mark.parametrize(
```

Comment:
I wasn't even aware of this!

Reply:
It's a really nice way to refactor the code and reduce the amount of repeated lines.